### PR TITLE
Hotseat gameplay

### PR DESCRIPTION
**Pull request by corez92**
Goal: get a working game play experience as if we're playing on one device.

### CODE REVIEW

File: tele_giphy/templates/base.html [hunk 1]
```diff
@@ -30,6 +30,9 @@
     <div class="row">
       <p class="navbar-text navbar-right"><a href="{% url 'game:index' %}" class="navbar-link"><button type="button" class="btn btn-default navbar-btn">Home </button></a></p>
     </div>
+    {% if error_message %}
+      <div class="alert alert-danger" role="alert"><strong>{{ error_message }}</strong></div>
+    {% endif %}
```
Comment (Mester) [inline tele_giphy/templates/base.html]: Put this in the messages block that we already have instead (line 42)

**Comment by corez92** [comment]
some zoomed out pictures of some of the game:
http://i.imgur.com/sowtYwT.png
http://i.imgur.com/mjjibsI.png

**Comment by corez92** [comment]
new layout: 
http://imgur.com/a/yEjCd

(buttons need to be rearranged/aligned properly and: space for user name and possibly timer should be added)

File: tele_giphy/game/models.py [hunk 2]
```diff
@@ -2,19 +2,34 @@
...
 # Keeps track of game rounds
-class GameRounds(models.Model):
+class GameRound(models.Model):
     round_number = models.IntegerField()
     user_text = models.CharField(max_length=150)
     giphy_url = models.CharField(max_length=2083) #Should be the AWS S3 URL, uses limitation of URL length
-    user = models.ForeignKey(User, on_delete=models.CASCADE)
-    game = models.ForeignKey(Game, on_delete=models.CASCADE)
\ No newline at end of file
+    # user = models.ForeignKey(User, on_delete=models.CASCADE) #many to many relationship instead of many to one relationship?
```
Comment (zapplecat) [inline tele_giphy/game/models.py:32]: See #43

**Pull request merged**